config/longrun_configs/longrun_aquaplanet_rhoe_equil_55km_nz63_clearsky_0M_earth.yml

```diff
@@ -1,23 +1,26 @@
+approximate_linear_solve_iters: 2
+divergence_damping_factor: 4.0
+dt: "150secs"
+dt_rad: "6hours"
 dt_save_state_to_disk: "10days"
-dt: "100secs"
-t_end: "200days"
-h_elem: 16
-z_elem: 63
 dz_bottom: 30.0
 dz_top: 3000.0
-z_max: 55000.0
-rayleigh_sponge: true
-topography: "Earth"
-topo_smoothing: true
-vorticity_hyperdiffusion_coefficient: 4.0
-scalar_hyperdiffusion_coefficient: 4.0
+h_elem: 16
+implicit_diffusion: true
+job_id: "longrun_aquaplanet_rhoe_equil_55km_nz63_clearsky_0M_earth"
 moist: "equil"
 precip_model: "0M"
+rad: "clearsky"
+rayleigh_sponge: true
+scalar_hyperdiffusion_coefficient: 4.0
+smoothing_order: 2
 surface_setup: "DefaultMoninObukhov"
+t_end: "900days"
```

Comment:
Does it run 900 days within 1 day (your build was a restarting run)?

Reply:
Yes; the restarted (500day) simulation took ~7h, I expect this will be done within 14h, we can bring it down to 6h if that's helpful. I'll revert the yaml groupings as well (maybe we want to revisit that ordering once the config files start to grow too large)

Reply:
T0, T1 are the jobs we wish to merge from this build: https://buildkite.com/clima/climaatmos-gpulongruns/builds/53#018deba0-d8f7-48ea-89b7-a21272bf6cb9